app/scripts/services/wsService.js

```diff
@@ -403,9 +403,13 @@ wsService.factory('dbXrefService', [
                     var match = _.find(xrefs, function (xref) {
                         return xref.database === name || _.contains(xref.synonyms, name);
                     });
-                    return match
-                        .entity_types[0]
-                        .url_syntax
+                    var urlpath = match.entity_types[0].url_syntax;
+                    var correctBeginning = "http://";
```

Comment:
Could `urlPrefix` or something like that be a better name than `correctBeginning`? Would make it easier for another developer to know what it is.